.github/actions/pr_notifier/pr_notifier.py

```diff
@@ -98,37 +129,38 @@ def track_prs():
         # SLO, nudge in bold if not.
         message = pr_message(delta, pr_info.html_url, pr_info.title, delta_days, delta_hours)
 
+        if (needs_api_review(labels, repo, pr_info)):
+            add_reminders(pr_info.assignees, api_review_and_prs, message, API_REVIEWERS)
```

Comment:
is it ever the case that an api PRs are unassigned? I'm pretty sure repokitteh handles that by random assignment anyway though.. so i think that's ok

Reply:
Yeah, it should always be assigned but we could arguably find it not having an API reviewer if someone removed themselves for vacation.  We could have a list for unassigned_api - I'll consider it as a feature if it ever happens or @envoyproxy/api-shepherds tell me it's worth it :-)